.github/workflows/ci.yml

```diff
@@ -26,11 +26,14 @@ jobs:
           key: ${{ runner.os }}-nuget-${{ hashFiles('**/packages.lock.json') }}
           restore-keys: |
             ${{ runner.os }}-nuget-
-
-      - name: Setup dotnet
+      - name: Setup dotnet 3.1
         uses: actions/setup-dotnet@v1
         with:
           dotnet-version: '3.1'
+      - name: Setup dotnet 5
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '5.0.100'
```

Comment:
related to the Last commit's, seems we can't go this way :/

Reply:
lets use `5.0` to avoid the quarterly releases, I tested it here: https://github.com/Specshell/specshell.software.ndde/pull/2
```suggestion
          dotnet-version: '5.0'
```

Reply:
We can't go with that see https://github.com/nmklotas/GitLabApiClient/pull/186/commits/9e10a9ba7100f43b6460bdfd5cbe9018967b01ad